config/fcos/v1_6_exp/translate.go

```diff
@@ -326,9 +299,6 @@ func (c Config) processBootDevice(config *types.Config, ts *translate.Translatio
 	case wantMirror:
 		// RAID without LUKS
 		rootDevice = "/dev/md/md-root"
-	case wantLuksDevice:
```

Comment:
So we added a case and then removed it. This makes reviewing this a bit confusing, this change should have been amended to the originating commit.